projects/fizzbuzz/C/AlexAckerlund/FizzBuzz.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+
+int main()
+{
+	int input = printf("Enter a number greater than 0: ");
```

Comment:
printf is returning the error 31 if a char is entered instead of an int. This could be easily fixed by declaring input without initialization and calling printf on the next line.